my-app/src/index.tsx

```diff
@@ -1,19 +1,44 @@
-import React from 'react';
-import ReactDOM from 'react-dom/client';
-import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import { css } from "@emotion/css";
+import Typography from "@mui/material/Typography";
```

Comment:
This is not imported as it should be. 

When you import from material, you are importing like this:
So everything is from the same @mui/material:

import { Box, Typography } from "@mui/material";